server/mem_agent/engine.py

```
@@ -0,0 +1,334 @@
+                    try:
+                        mod_obj = importlib.import_module(mod_name)
+                    except ImportError:
+                        mod_obj = None
+                    # If module is imported in sandbox, remove the attribute
+                    if mod_obj and hasattr(mod_obj, attr_name):
+                        try:
+                            setattr(
+                                mod_obj, attr_name, None
+                            )  # simple way: nullify the attribute
+                        except Exception:
+                            pass  # if we cannot set it, ignore (might be read-only)
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)